internal/pkg/cli/app_delete.go

```diff
@@ -58,6 +58,7 @@ type deleteAppOpts struct {
 	prompt               prompter
 	s3                   func(session *session.Session) bucketEmptier
 	executor             func(svcName string) (executor, error)
```

Comment:
can we update this field to be `svcExecutor` so that it's consistent with `jobExecutor`?
Or maybe even better both to: `svcDeleteExecutor` and `jobDeleteExecutor`

Reply:
Yes. Also changed askExecutor to envDeleteExecutor.